packages/cli/lib/serverlessLogs.js

```diff
@@ -63,6 +63,7 @@ const tailLogs = async ({
           })
         );
       }
+      process.exit();
```

Comment:
Should we provide some feedback to the user that we have cancelled the polling?

Reply:
I added a `logger.info` message "Stopped polling due to error", and the error should also be logged.